### PR TITLE
DHFPROD-2335: Use source/target labeling for ingestion config

### PR DESCRIPTION
**Pull request by wooldridge**
Change input/output terms in labels to source/target for ingestion configuration:

- Source File Path
- Source File Type
- Target File Type
- Target Permissions
- Target URI Replacement

Additional related fixes:
- Add title attribute to Source value in step
- Change ingestion step label in step from “Source” to "Source File Path"

### CODE REVIEW

**Comment by ayuwono** [comment]
@wooldridge , did you change the selectors as well? Or they are still using the old names?

**Comment by wooldridge** [comment]
@ayuwono good catch. Are you OK with me updating those as well in this PR?

**Comment by ayuwono** [comment]
@wooldridge, If you change it here, it will make the UI tests failed. I think it's better to change it here as there are not too many fields on ingestion step. Once you make a change, I need to update the ingestion page objects: https://github.com/marklogic/marklogic-data-hub/blob/bugfix/DHFPROD-2335-ingestion-labels/web/e2e/page-objects/steps/ingestStep.ts

**Comment by wooldridge** [comment]
@ayuwono I updated the selector names, you should take a look to see if they are OK.

File: web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html [hunk 1]
```diff
@@ -17,15 +17,15 @@
     <div class="label">{{config.fileTypes.label}}</div>
     <mat-form-field layout-fill appearance="outline">
       <mat-select
-        id="file_type_select"
+        id="source-type-select"
```
Comment (ayuwono) [inline web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html]: should we be consistent with the label name? source-file-type-select is a better id. It's easier to identify as page object

File: web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
         [(ngModel)]="this.step.fileLocations.inputFileType"
         name="filetype"
         (selectionChange)="onChange()"
       >
         <mat-option
           *ngFor="let option of config.fileTypes.options"
           [value]="option.value"
-          [id]="'file-type-option-' + option.value"
+          [id]="'source-type-option-' + option.value"
```
Comment (ayuwono) [inline web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html]: source-file-type-option

File: web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html [hunk 3]
```diff
@@ -34,15 +34,15 @@
     <div class="label">{{config.outputDocTypes.label}}</div>
     <mat-form-field layout-fill appearance="outline">
       <mat-select
-        id="doc_type_select"
+        id="target-type-select"
```
Comment (ayuwono) [inline web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html]: target-file-type-select

File: web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html [hunk 4]
```diff
@@ -34,15 +34,15 @@
...
         [(ngModel)]="this.step.options.outputFormat"
         name="doctype"
         (selectionChange)="onChange()"
       >
         <mat-option
           *ngFor="let option of config.outputDocTypes.options"
           [value]="option.value"
-          [id]="'doc-type-option-' + option.value"
+          [id]="'target-type-option-' + option.value"
```
Comment (ayuwono) [inline web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html]: target-file-type-option

File: web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts [hunk 5]
```diff
@@ -2,12 +2,12 @@ import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
 
 const settings = {
   inputFilePath: {
-    label: 'Input File Path',
+    label: 'Source File Path',
```
Comment (ayuwono) [inline web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts]: we need to change the selector for this element. Right now it's a class folder-path. We should change it to source-file-path

**Comment by clockworked247** [comment]
@wooldridge please update the master selector list too:  
https://wiki.marklogic.com/display/ENGINEERING/QuickStart+UI+Selectors+for+Testing

**Comment by clockworked247** [comment]
Will approve after @ayuwono suggested changes are made.

**Comment by wooldridge** [comment]
@ayuwono addressed your requests with latest commits. @clockworked247 also updated wiki page.

**Pull request merged**